### DISCUSSION TITLE
Web scrapping (IMDB) "By Metacritic" tab chart not working

### DISCUSSION THREAD

**Discussion by lhpratama** [teamalgoritma/community#762] [Q&A] [answered]
Tab pada bagian "By Metascore" menunjukkan chart tab "By Votes". Sedangkan, kedua tab lainnya sudah berhasil 

![Error By Metascore](https://github.com/teamalgoritma/community/assets/141613272/42be6ec1-b728-4643-aacc-333c3d1225a9)

code pada app.py :

![apppy capstone](https://github.com/teamalgoritma/community/assets/141613272/c970f235-a96e-4272-8882-67caeced9978)

code pada index.html :

![indexhtml capstone](https://github.com/teamalgoritma/community/assets/141613272/e1e055af-80e4-40fe-8022-04c5147cb448)

**Comment by chchlts** [comment, marked as answer]
Dear @lhpratama 

Untuk menampilkan plot yang berbeda untuk setiap tab. Bisa merujuk ke discussion https://github.com/teamalgoritma/community/discussions/759. 

Semoga membantu

Best Regards,
Team Algoritma
